internal/rukpak/source/containers_image.go

```diff
@@ -171,6 +194,35 @@ func (i *ContainersImageRegistry) Cleanup(_ context.Context, bundle *BundleSourc
 	return deleteRecursive(i.bundlePath(bundle.Name))
 }
 
+func (i *ContainersImageRegistry) checkUpdatedConfiguration() error {
+	i.checkConfigLock.Lock()
+	defer i.checkConfigLock.Unlock()
+	dirEntries, err := os.ReadDir(containersConfigFolder)
+	if err != nil && !os.IsNotExist(err) {
+		// Ignore not found errors
+		return fmt.Errorf("could not read registries configuration directory, error: %w", err)
+	}
+	for _, entry := range dirEntries {
+		if entry.Type().IsRegular() || !strings.HasPrefix(entry.Name(), "..") {
+			// Skip 'normal' entries
+			continue
+		}
+		// Trim the leading '..' for symlinks
+		trimmed := strings.TrimLeft(entry.Name(), "..")
```

Comment:
`TrimLeft` would actually strip all of the leading `.` chars instead of `..`. You might use `TrimPrefix` instead.